src/main/java/com/savvato/tribeapp/controllers/ReviewDecisionAPIController.java

```diff
@@ -12,24 +15,39 @@
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import java.util.List;
+import java.util.Optional;
+
 @RestController
 @Tag(name = "reviewer-decision", description = "Decisions made after reviewing an attribute")
 @RequestMapping("/api/reviewer-decision")
 public class ReviewDecisionAPIController {
-  @Autowired ReviewDecisionService reviewDecisionService;
+  @Autowired 
+  ReviewDecisionService reviewDecisionService;
+
+  @Autowired 
+  ReviewDecisionReasonService reviewDecisionReasonService;
 
   @SaveReviewDecision
   @PostMapping
-  public ResponseEntity<ReviewDecisionDTO> saveReviewDecision(
-      @RequestBody @Valid ReviewDecisionRequest request) {
+  public ResponseEntity<ReviewDecisionDTO> saveReviewDecision(@RequestBody @Valid ReviewDecisionRequest request) {
     ReviewDecision decisionSaved =
-        reviewDecisionService.saveReviewDecision(
-            request.reviewId, request.reviewerId, request.reasonId);
+        reviewDecisionService.saveReviewDecision(request.reviewId, request.userId, request.reasonId);
     ReviewDecisionDTO rtn = ReviewDecisionDTO.builder().build();
 
     rtn.reviewId = decisionSaved.getReviewId();
     rtn.userId = decisionSaved.getUserId();
     rtn.reasonId = decisionSaved.getReasonId();
     return ResponseEntity.status(HttpStatus.OK).body(rtn);
   }
+
+  @GetMapping
+  public ResponseEntity getReasonList() {
+    List<ReviewDecisionReasonDTO> reasonsList = reviewDecisionReasonService.getReviewDecisionReasons();
+    if (!reasonsList.isEmpty()) {
```

Comment:
My recommendation when a list is empty is to return an empty list, in the front end just show a list empty and not receive an error because the list is empty.

Reply:
@fontolangustavo Usually, I would agree with you. In this case, if there are no Review Decision Reasons, then our server/database is in a bad state. Perhaps we could return a better error than BAD_REQUEST, but we certainly should return an error.

This PR is old, the guy who was working on it no longer attends. If you'd like to clean it up, and get it pushed through, that'd be appreciated.